storage/blockdevice/COMPONENT_OSPIF/include/OSPIF/OSPIFBlockDevice.h

```diff
@@ -327,7 +327,9 @@ class OSPIFBlockDevice : public mbed::BlockDevice {
                                              mbed::bd_size_t tx_length, const char *rx_buffer, mbed::bd_size_t rx_length);
 
     // Send command to read from the SFDP table
-    int _ospi_send_read_sfdp_command(mbed::bd_addr_t addr, void *rx_buffer, mbed::bd_size_t rx_length);
+    int _ospi_send_read_sfdp_command(mbed::bd_addr_t addr, mbed::sfdp_cmd_addr_size_t addr_size,
```

Comment:
Same goes for these other callbacks (should be part of "SFDP: Add more parameters to the reader callback")